full-stack-tests/imodelhub-client/src/imodelhub/TestUtils.ts

```diff
@@ -964,20 +951,6 @@ export class ProgressTracker {
 
 let cloudEnv: IModelCloudEnvironment | undefined;
 
-if (!TestConfig.enableIModelBank || TestConfig.enableMocks) {
-  cloudEnv = new TestIModelHubCloudEnv();
-} else {
-  cloudEnv = getIModelBankCloudEnv();
-  imodelBankClient = cloudEnv.imodelClient as IModelBankClient;
-}
-if (cloudEnv === undefined)
-  throw new Error("could not create cloudEnv");
-
-cloudEnv.startup()
-  .catch((err: Error) => {
-    Logger.logException(loggingCategory, err);
-  });
-
 // TRICKY! All of the "describe" functions are called first. Many of them call getCloudEnv,

```

Comment:
Since you moved IModelClient creation into before functions, rather than describes, maybe these comments should be removed? I assume getCloudEnv isn't called anymore (at least until the following before launches).

Reply:
Mind if I do this in a separate PR? I'd like to get this one merged and I'll follow up with additional clean up